portfolio/src/main/webapp/script.js

```diff
@@ -59,4 +82,32 @@ async function loadProjects(){
   for(var i =0; i<projects.length; i++){
       projectList.innerHTML += "<li> <a target= \"_blank\" href=\""+projects[i].url+"\">"+projects[i].name;
   }
-}
\ No newline at end of file
+}
+
+function requestTranslation() {
+        const text = document.querySelectorAll('.txt');
+        const languageCode = document.getElementById('language').value;
+        console.log(text);
+        
+
+        
+        for(var i = 0; i<text.length; i++){
```

Comment:
You might consider using a for of loop here. Something like:

```
for (const textElement of textElements) {
  ...
  params.append('text', textElement.innerText);
  ...
}
```

(I'd also consider renaming `text` to something like `textElements` to be more descriptive.)